contributors/design-proposals/hugepages.md

```diff
@@ -125,7 +166,41 @@ pages, and Quota should be able to count them.
 
 Get design approval
 
-### Phase 1: Add huge page support
+### Phase 1: Huge page volume plugin
+
+**Target 1.8+**
+
+Implement huge page volume plugin. Accounting and limiting huge pages before 
+`Phase 2` can be done via OIR(opaque integer resources). 
```

Comment:
Will OIR be deprecated in favor of resource classes @vishh ? If so, is it ok to continue leveraging this for alpha features?